api_contracts/contracts/agency/AgencyProfile.md

```diff
@@ -0,0 +1,162 @@
+        The fiscal year that you are querying data for.
+    + funding_agency_id: 456 (required, number)
+        The unique USAspending.gov agency identifier. This ID is the `agency_id` value returned in the `/api/v2/references/toptier_agencies/` endpoint.
+
+### Get Financial Balances [GET]
+
++ Response 200 (application/json)
+    + Attributes
+        + results (required, array[FinancialBalance], fixed-type)
+
+## Major Object Classes [/api/v2/financial_spending/major_object_class/{?fiscal_year,funding_agency_id}]
+
+This endpoint returns the total amount that a specific agency has obligated to major object classes in a given fiscal year.
+
++ Parameters
+    + fiscal_year: 2017 (required, number)
+        The fiscal year that you are querying data for.
+    + funding_agency_id: 456 (required, number)
+        The unique USAspending.gov agency identifier. This ID is the `agency_id` value returned in the `/api/v2/references/toptier_agencies/` endpoint.
+
+### Get Major Object Classes [GET]
+
++ Response 200 (application/json)
+    + Attributes
+        + results (required, array[MajorObjectClass], fixed-type)
+
+## Major Object Classes [/api/v2/financial_spending/object_class/{?fiscal_year,funding_agency_id,major_object_class_code}]
+
+This endpoint returns the total amount that a specific agency has obligated to minor object classes within a specific major object class in a given fiscal year.
+
++ Parameters
+    + fiscal_year: 2017 (required, number)
+        The fiscal year that you are querying data for.
+    + funding_agency_id: 456 (required, number)
+        The unique USAspending.gov agency identifier. This ID is the `agency_id` value returned in the `/api/v2/references/toptier_agencies/` endpoint.
+    + major_object_class_code: 30 (required, number)
+        The major object class code returned in `/api/v2/financial_spending/major_object_class/`.
+
+### Get Minor Object Classes [GET]
+
++ Response 200 (application/json)
+    + Attributes
+        + results (required, array[MinorObjectClass], fixed-type)
+
+## Federal Accounts [/api/v2/federal_obligations/{?fiscal_year,funding_agency_id,limit,page}]
+
+This endpoint returns the amount that the specific agency has obligated to various federal accounts in a given fiscal year.
+
++ Parameters
+    + fiscal_year: 2017 (required, number)
+        The fiscal year that you are querying data for.
+    + funding_agency_id: 456 (required, number)
+        The unique USAspending.gov agency identifier. This ID is the `agency_id` value returned in the `/api/v2/references/toptier_agencies/` endpoint.
+    + limit: 10 (optional, number)
+        The maximum number of results to return in the response.
+    + page: 1 (optional, number)
+        The response page to return (the record offset is (`page` - 1) * `limit`).
+
+### Get Federal Accounts [GET]
+
++ Response 200 (application/json)
+    + Attributes
+        + results (required, array[FederalAccount], fixed-type)
```

Comment:
Missing `page_metadata`

Reply:
We decided to exclude the API elements that we weren't using (if another consumer were to use it, they would include it in their contract).